### ISSUE TITLE
Ignore and redirect external links

### ISSUE THREAD

**Issue by mythz** [lukeed/navaid#3]
This library takes over external links so when you click on an external link:

```html
<a href="http://google.com">link</a>
```

It instead redirects to:

    http://localhost:5000/http://google.com

My current workaround is detect an external link and replace the location like:

```js
if (u.indexOf('/http') === 0) {
    location.replace(u.substring(1))
    return;
}
```

But it still temporarily shows the invalid URL in the browser before redirecting and you can't use `location.href = u.substring(1)` because it adds the invalid URL to the browser history so when you go back it goes back to the URL.

IMO external links should not be caught and redirected immediately, or if you don't agree with that default behavior can you provide an option to not handle external links. 

Alternatively it would be more flexible if there was a callback where we can intercept the location change so we can immediately redirect ourselves using `location.href`.

**Comment by lukeed** [comment]
Noticed this today as well. I typically have `target=""` on external links, so I've avoided this issue thus far.

Will fix it asap

**Comment by mythz** [comment]
Thanks! Can you publish an update to npm when you have a chance, Cheers 👍

**Comment by lukeed** [comment]
Released now as `1.0.1`. 

I was seeing if I could squeeze out any additional byte shavings... I did 😇 Took it from what would have been 846B to 829B. Free feature
